<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLinkChecker</class>
 <widget class="QDialog" name="DialogLinkChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tvLink"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cbAddToQueue">
     <property name="toolTip">
      <string>Add all link to download Queue.</string>
     </property>
     <property name="text">
      <string>Add to Downloads Queue</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save Files to:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="saveTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pbBrowse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbStart">
     <property name="text">
      <string>Start Checking</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbAddToQueue</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAddToQueue</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAddToQueue</sender>
   <signal>toggled(bool)</signal>
   <receiver>pbBrowse</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
